zoo-css-challenge/README.md

```diff
@@ -38,30 +34,26 @@ Change the background colour to #483C46, the section titles to #F4743B, and the
 
 Make the first paragraph have a larger font size.
 
-Again, the purple headings are hard to read on dark backgrounds, change them to a lighter colour.
-
 ## 5) News
 
 Can the title of this section be centred in the middle of this page?
 
 Add another news item with the title with today's date, and the title of ‘Which big cat are you?’
 
-## 6) Membership
+## 6) Learning
```

Comment:
The `Learning` section was moved from below as it comes first in the source code. Having it in a different order confused me when going through the tasks